### DISCUSSION TITLE
Migration from personal account to organization!

### DISCUSSION THREAD

**Discussion by surya-prakash-susarla** [NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11#38] [Announcements]
Hey everyone!

We've moved from my the personal account where we initially forked the repo (my acount - Surya-06) to a new organization which I have created specifically for our project for this semester. Everyone has full access over all parts of the repo and I have sent out invites for adding folks as admins of the org.

Thanks!

**Comment by kaushikjadhav01** [comment]
Great, thanks! Updated org name, basic details and profile picture!
